src/pages/resources/videos/exploring/projects-and-workspaces.md

```diff
@@ -1,16 +1,16 @@
 ---
-title: Exploring Project Firefly - Projects and Workspaces
-description: The Project Firefly team shared a short presentation about projects and workspaces and how to use customizations to help your team collaborate on building Firefly Applications.  
+title: Exploring App Builder - Projects and Workspaces
+description: The App Builder team shared a short presentation about projects and workspaces and how to use customizations to help your team collaborate on building Firefly Applications.  
 keywords:
   - Adobe I/O
   - Extensibility
   - API Documentation
   - Developer Tooling  
 ---
 
-# Exploring Project Firefly - Projects and Workspaces
+# Exploring App Builder - Projects and Workspaces
 
-The Project Firefly team shared a short presentation about projects and workspaces and how to use customizations to help your team collaborate on building Firefly Applications.
+The App Builder team shared a short presentation about projects and workspaces and how to use customizations to help your team collaborate on building Firefly Applications.
```

Comment:
missed this reference "to help your team collaborate on building Firefly Applications."